fibbingnode/res/templates/ospf.mako

```diff
@@ -35,4 +35,9 @@ router ospf
   % for itf in node.ospf.passive_interfaces:
   passive-interface ${itf.name}
   % endfor
+  % if node.ospf.throttling and node.ospf.lsa:
+  timers throttle spf ${node.ospf.throttling['spf'].get('delay')} ${node.ospf.throttling['spf'].get('initial_holdtime')} ${node.ospf.throttling['spf'].get('max_holdtime')}
```

Comment:
Why do you need those get() calls ? especially since you don't provide the default value if the key is not found as the config line will then be 'timers throttle spf None ...'